puppetboard/app.py

```diff
@@ -1031,8 +1031,13 @@ def catalog_compare(env, compare, against):
 
 
 @app.route('/radiator', defaults={'env': app.config['DEFAULT_ENVIRONMENT']})
+@app.route('/radiator.json', endpoint='radiator_json',
+           defaults={'env': app.config['DEFAULT_ENVIRONMENT'],
+                     'to_json': True})
+@app.route('/<env>/radiator.json', endpoint='env_radiator_json',
+           defaults={'to_json': True})
 @app.route('/<env>/radiator')
-def radiator(env):
+def radiator(env, to_json=False):
```

Comment:
Instead of adding a to_json lets use the Accept header:

    if ('Accept' in request.headers
        and request.headers["Accept"] == 'application/json'):
        return jsonify(**stats)